src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -37,6 +37,9 @@
 					</template>
 				</kv-lightbox>
 			</div>
+			<div>
+				<loan-search-filter-chips :loan-search-state="loanSearchState" :all-facets="allFacets" />
```

Comment:
You'll need to add `@updated="handleUpdatedFilters"` to this instance and ensure the data associated with the `updated` event emitted from LoanSearchFilterChips.vue is formatted correctly and contains the correct data fields.